editor/editor_inspector.cpp

```diff
@@ -1267,6 +1295,15 @@ void EditorInspectorCategory::_notification(int p_what) {
 	}
 }
 
+void EditorInspectorCategory::override_menu(PopupMenu *p_menu) {
+	if (!p_menu) {
+		menu_override = nullptr;
+		return;
+	}
```

Comment:
??

Reply:
I was probably thinking about adding more stuff here, didn't need to, but also forgot the clean up. Brain fart on my end.